examples/work.yaml

```diff
@@ -147,10 +147,18 @@
       qudt:unit: unit:Hour
       qudt:numericValue: 21.5
     observedTime:
+      '@type': time:Instant
+      time:inXSDDateTimeStamp: 2018-10-31T00:00:00-5:00
+    accountDuring: org:9087abc4-6f2b-4f1f-9c56-611b7f7302a2
+
+  - @id': org:9087abc4-6f2b-4f1f-9c56-611b7f7302a2
+    '@type': AccountingPeriod
```

Comment:
Let's see if we can't just use 'upper' level Process in *composition* milestone. Possibly some work, usage maybe even consumption needs to recorded for that 'accounting period' and representing it as Process would make more sense.

Reply:
Hmmm, I don't think of it as a Process, but I'm fine waiting until that milestone to define it.  It does fit with the budgeting stuff from the SolidBase crew.  In addition to this operational usage.  Maybe it is often more of an operational period than an accounting period... ?

Thoughts about how to represent the time property in the FairCoop facilitation UC and the buying club UC?

Reply:
I'll take a look at UC which provides facilitation services based on work and usage of computer, workspace etc. In practice many remote work services require more than just raw work and dependent of usage of various other resources.

Reply:
In these UCs, a service wouldn't work, it is basically people, either getting paid by the hour or getting buying club privileges in exchange for worked hours.  We can not worry about it now.  My question was more about how to represent the time if it isn't the duration of the accounting/operational period?

Reply:
The main challenge I see comes with a fact that we don't try to represent an actual *work* interval that occurred at specific time. It seems more like a report based on not recorded events which includes sum from those unrecorded events. In a way a duration of non continuous interval.

>  My question was more about how to represent the time

If we just need some time interval we can always use instances of classes from OWL-Time, for example `time:ProperInterval`. I think VF only needs to focus on what do those time entity mean for the economic environment and what their relations with other entities mean.

Reply:
I'm totally fine with using the ProperInterval or whatever and it being a month or whatever, but you didn't like that too much.  It then only relates to the timing that matters to them, and of course doesn't represent the hours actually worked.  Often people also only care about due date, or start date, for example.  I guess the question is do we leave it so people can do what they want for recording time, depending on how much coordination they need?  Or do we define that it must be the actual duration used or worked?  I support the first, based on what people actually do.  I don't see any down side from the network standpoint, or larger planning or anything.  As long as the quantities are there as needed.  It just says this quantity happened sometime(s) during this time.

I do understand your challenge, we had the same first response and tried to recommend entering all actual time.  But it actually doesn't make sense for some kinds of work - sometimes they are meant to be around the right amount, but work more like "salary" than "wages".  Sometimes they do write it down and just record the sum (I do that for the buying club); and sometimes the work is like a few minutes here and there, and doesn't make sense to record exactly.